Makefile

```diff
@@ -8,6 +8,12 @@ pkg:
 	@echo "==> $@"
 	$(PYTHON) -m build .
 
+.PHONY: update
+update:
+	@echo "==> $@"
+	@rm -rf ./deps
```

Comment:
Update was using `deps` as a semi-permanent temporary directory to avoid retrieving everything everytime. If we are going to remove it every time, maybe we should just update the script to download everything to a temporary directory instead.